pkg/autoscaler/metrics/metrics_provider.go

```diff
@@ -21,31 +21,55 @@ import (
 	"math"
 	"time"
 
-	"github.com/google/go-cmp/cmp"
 	"github.com/kubernetes-incubator/custom-metrics-apiserver/pkg/provider"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
 	cmetrics "k8s.io/metrics/pkg/apis/custom_metrics"
 	"knative.dev/serving/pkg/apis/autoscaling"
+	v1 "knative.dev/serving/pkg/apis/serving/v1"
 	"knative.dev/serving/pkg/apis/serving/v1alpha1"
 )
 
+type getMetric func(MetricClient, types.NamespacedName, time.Time) (float64, float64, error)
+
 var (
 	concurrencyMetricInfo = provider.CustomMetricInfo{
-		GroupResource: v1alpha1.Resource("revisions"),
+		GroupResource: v1.Resource("revisions"),
 		Namespaced:    true,
 		Metric:        autoscaling.Concurrency,
 	}
+
 	rpsMetricInfo = provider.CustomMetricInfo{
-		GroupResource: v1alpha1.Resource("revisions"),
+		GroupResource: v1.Resource("revisions"),
 		Namespaced:    true,
 		Metric:        autoscaling.RPS,
 	}
 
 	errMetricNotSupported = errors.New("metric not supported")
 	errNotImplemented     = errors.New("not implemented")
+
+	deprecatedConcurrencyMetricInfo = provider.CustomMetricInfo{
```

Comment:
Let's add a TODO here, to remove these once 0.13 cuts.

Reply:
done